torchvision/ops/_custom_ops.py

```diff
@@ -0,0 +1,44 @@
+import os
+import torch
+import glob
+
+# load the custom_op_library and register the custom ops
+custom_op_lib_path = os.path.join(os.path.dirname(__file__), '..', 'custom_ops.*.so')
+custom_op_lib = glob.glob(custom_op_lib_path)
+torch.ops.load_library(custom_op_lib[0])
```

Comment:
Can there be a situation where multiple files are found?

Reply:
Personally, I think it would be cleaner to do the glob at build time and rename to `custom_ops.so|dylib|dll` then. At build time throwing an error when seeing two seems prudent. This way, we don't need glob at runtime.